### ISSUE TITLE
Make all other elements match the height of a specified element

### ISSUE THREAD

**Issue by afmab** [liabru/jquery-match-height#59]
I have a requirement where I need to be able to specify a target element, and then rather than all elements matching the row's maximum height, they will match the target's height.

For now I have added a ```data-mh-target``` attribute to the relevant element and then modified the iteration at line 197 so that it does the following, which seems to work okay.

```javascript
$.each(rows, function(key, row) {
            var $row = $(row),
                maxHeight = 0,
				targetExists = false;

            // skip apply to rows with only one item
            if (opts.byRow && $row.length <= 1) {
                $row.css(opts.property, '');
                return;
            }
			
            // iterate the row and find the max height
	        $row.each(function() {
		        if (targetExists)
		        	return;
				
                var $that = $(this),
                    display = $that.css('display') === 'inline-block' ? 'inline-block' : 'block';

                // ensure we get the correct actual height (and not a previously set height value)
                var css = { 'display': display };
                css[opts.property] = '';
                $that.css(css);

		        targetExists = $that.is("[data-mh-target]");

                // find the max height (including padding, but not margin)
		        if (targetExists || $that.outerHeight(false) > maxHeight)
                    maxHeight = $that.outerHeight(false);

                // revert display block
                $that.css('display', '');
            });
```

Please can you confirm whether there's an existing way for me to achieve the above, or if not whether the functionality above could be added?

Thanks

**Comment by liabru** [comment]
This sounds like a useful suggestion. I'll have a play with the idea, thanks!

**Comment by liabru** [comment]
This has now been implemented (as an option, not a data api) in 9239f4e if you'd like to try it.

**Comment by afmab** [comment]
I've tested the latest version and can confirm it's working. Thanks for your speedy response, and thanks for the plugin - very handy!
